### ISSUE TITLE
Connector for deezer is not working

### ISSUE THREAD

**Issue by JunkieJunkie** [web-scrobbler/web-scrobbler#4078]
**Describe the bug**
I hit play on music but web scrobbler says no music is playing

It started to stop working a few hours ago, before it was going normal

**Screenshots**
![image](https://github.com/web-scrobbler/web-scrobbler/assets/144092279/11db0d6a-4693-4150-bfce-8aa90ccf2ca3)

**Environment (please complete the following information)**
Windows 11
Opera Gx
The Web Scrobbler version is v3.1.0

**Comment by Sherer420** [comment]
+1
Ignores music from deezer

Windows 10
Google Chrome
Web Scrobbler v3.1.0

**Comment by yayuyokitano** [comment]
Can reproduce. Seems like deezer has changed something on their website.
Think I should have time to look into it today.

**Comment by mhall-tamro** [comment]
+1
I fixed by uninstalling Chrome Store version and installing Edge Store version (3.0.14). In the title bar, I regained green note / checkmark instead of the ws symbol too. So, I think this will be a clue to fixing the problem. (It probably was due to change to Chrome Store version, not change to Deezer).

Windows 11
Microsoft Edge
Chrome Store version -> Edge Store version 3.0.14

**Comment by MatF2F** [comment]
+1
Windows 10
Brave
Web Scrobbler v3.1.0

**Comment by yayuyokitano** [comment]
> +1 I fixed by uninstalling Chrome Store version and installing Edge Store version (3.0.14). In the title bar, I regained green note / checkmark instead of the ws symbol too. So, I think this will be a clue to fixing the problem. (It probably was due to change to Chrome Store version, not change to Deezer).
> 
> Windows 11 Microsoft Edge Chrome Store version -> Edge Store version 3.0.14

Odd, in my initial testing i seemed to have the issue occur on 3.0. Thank you for the info, will keep in mind when looking into it closer.

**Comment by yayuyokitano** [comment]
Found the issue. It's a bit hard to debug for me though, as web scrobbler just doesn't seem to be working on deezer for any version. (maybe it's because I'm using free?)
Anyway, it seems to be the imports in #3975.
Connectors cannot have imports, those break due to how connectors are loaded.
That should have been caught.
It's too late for me to actually fix the issue today personally though, but I'm pretty confident removing import is all there is to the issue.

**Comment by JunkieJunkie** [comment]
> Found the issue. It's a bit hard to debug for me though, as web scrobbler just doesn't seem to be working on deezer for any version. (maybe it's because I'm using free?) Anyway, it seems to be the imports in #3975. Connectors cannot have imports, those break due to how connectors are loaded. That should have been caught. It's too late for me to actually fix the issue today personally though, but I'm pretty confident removing import is all there is to the issue. EDIT: I actually think just replacing import with import type and doing nothing else might fix it.

I waiting you fix the problem, no rush i think it will right

**Comment by themidgarzolom** [comment]
@yayuyokitano Hi, I see this is closed.  I've had the same problem since the afternoon of Sept 4th (when my album stream randomly became disconnected about halfway through).  I made the GitHub account just to ask because I've been following Junkie's report and have the same issue, but Deezer connector is unfortunately **still not working for me** after your fix report.  My account is logged in (both with the chrome extension and Deezer) and functioning in every other way.  The Web Scrobbler works fine if I test it on YouTube with that connector.

Any help would be so appreciated, but mostly I just wanted to alert that this is still a malfunction for at least one user.

**Comment by inverse** [comment]
@themidgarzolom the issue is fixed in the repo but a new version isn't published yet.

**Comment by themidgarzolom** [comment]
@inverse So for someone like myself who doesn't know how any of this works, does that basically mean in layman's terms that the code is corrected but just not updated for public use with the extension yet and basically just sit tight for now?

**Comment by inverse** [comment]
Correct - we should publish the fix soon as it's a regression in the latest release.

**Comment by themidgarzolom** [comment]
@inverse Okay, thanks.
